src/library-authoring/data/apiHook.js

```diff
@@ -17,6 +30,22 @@ export const useContentLibrary = (libraryId) => (
   })
 );
 
+/**
+ * Use this mutation to create a block in a library
+ * @param {string} [libraryId]
+ */
+export const useCreateLibraryBlock = (libraryId) => {
+  const queryClient = useQueryClient();
+  return useMutation({
+    /** @type {import("@tanstack/react-query").MutateFunction<any, any, CreateBlockData>} */
+    mutationFn: async (data) => createLibraryBlock(data),
```

Comment:
The following will work, and you won't need to explicitly type it.

```suggestion
    mutationFn: createLibraryBlock,
```